/*ERROR: LINEA 6 | COLUMNA 5 | Se esperaba: id_objeto, se encontr√≥: p_main*/

class Prueba {


fn main () {

}

}